airbyte-integrations/connectors/source-mixpanel/unit_tests/conftest.py

```diff
@@ -0,0 +1,190 @@
+import pendulum
+import pytest
+
+
```

Comment:
`conftest.py` is generally used to store fixtures that are reused by multiple test modules. I think these fixtures can be directly defined in `test_source`and `test_stream`.

Reply:
To be honest I prefer to keep test classes clean of any mock responses and dto objects. Should we maybe move it to a different file i.e mockresponse.py?

Reply:
I have went with your suggestions and I have moved the fixtures in the test classes where they are used :)

Reply:
Thanks, could you please declare the fixtures **before** the tests where they are used.